src/components/TodoList.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```

Comment:
As of React v16.13 or 17, the React import is not required

Reply:
Removing this line resolves into an error in the console saying React is not defined.

Reply:
Are you using React methods ( React.Children | Fragment  ...)?
If you have some <React.Something/> component or method, that might be the reason. Otherwise not possible.